sdk/nonce/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "solana-nonce"
+description = "Solana durable transaction nonces."
+documentation = "https://docs.rs/solana-nonce"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[dependencies]
+serde = { workspace = true, optional = true }
+serde_derive = { workspace = true, optional = true }
+solana-fee-calculator = { workspace = true }
+solana-hash = { workspace = true, default-features = false }
+solana-pubkey = { workspace = true, default-features = false }
+solana-sha256-hasher = { workspace = true }
+
+[dev-dependencies]
+bincode = { workspace = true }
+solana-nonce = { path = ".", features = ["dev-context-only-utils"] }
+
+[features]
+dev-context-only-utils = ["serde"]
+serde = [
+    "dep:serde",
+    "dep:serde_derive",
+    "solana-fee-calculator/serde",
+    "solana-hash/serde",
+    "solana-pubkey/serde",
+]
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
```

Wanna add the features bit and `#![cfg_attr(docsrs, feature(doc_auto_cfg))]`?

```suggestion
[package.metadata.docs.rs]
targets = ["x86_64-unknown-linux-gnu"]
all-features = true
rustdoc-args = ["--cfg=docsrs"]
```